app/feature/feature-edit-coinsured/src/main/kotlin/com/hedvig/android/feature/editcoinsured/ui/EditCoInsuredAddOrRemoveDestination.kt

```diff
@@ -117,6 +124,11 @@ private fun EditCoInsuredScreen(
 
       is EditCoInsuredState.Loaded -> {
         val coroutineScope = rememberCoroutineScope()
+
+        if (uiState.contractUpdateDate != null) {
+          onCompleted(uiState.contractUpdateDate)
+        }
```

Comment:
This may trigger more than once since we're doing it in composition. Consider doing it inside a `LaunchedEffect` for example